tests/suites/0_stateless/09_fuse_engine/09_0011_create_table_with_opt_loc.sh

```diff
@@ -11,12 +11,14 @@ echo "show value of table being cloned"
 echo "select *  from t09_0011" | $MYSQL_CLIENT_CONNECT
 
 ## get the snapshot id
-SNAPSHOT_LOC=$(echo "show create table default.t09_0011" | mysql -h127.0.0.1 -P3307 -uroot -s | sed 's/.*SNAPSHOT_LOC=\(.*\).*/\1/g')
+SNAPSHOT_LOC=$(echo "select location from fuse_history('default','t09_0011')" | mysql -h127.0.0.1 -P3307 -uroot -s)
+
 ## create a shallow clones of t09_0011 by using the table option 'snapshot_loc'
 ## using lower case option key
-echo "create table t09_0011_clone1(c int) snapshot_loc=$SNAPSHOT_LOC" | $MYSQL_CLIENT_CONNECT
+echo "create table t09_0011_clone1(c int) snapshot_loc='$SNAPSHOT_LOC'" | $MYSQL_CLIENT_CONNECT
 ## using upper case option key
-echo "create table t09_0011_clone2(c int) SNAPSHOT_LOC=$SNAPSHOT_LOC" | $MYSQL_CLIENT_CONNECT
+echo "create table t09_0011_clone2(c int) SNAPSHOT_LOC='$SNAPSHOT_LOC'" | $MYSQL_CLIENT_CONNECT
```

Comment:
Can we use the full name:
SNAPSHOT_LOC -> SNAPSHOT_LOCATION?

Let's make user not confused at 'LOC'.

Reply:
OK